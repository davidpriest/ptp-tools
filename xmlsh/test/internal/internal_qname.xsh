# 
# Tests of QName function
set +indent

echo QName(http://foo.bar spam)
echo QName(prefix http://foo.bar spam)
echo QName(foo "" spam)



a=QName(http://foo.bar bar)
xtype $a
echo <[ element {$a}{"hi"} ]>
a=QName(prefix1 http://foo.bar1 bar)
b=QName(prefix2 http://foo.bar2 spam)
echo <[ element {$b} { attribute {$a} { "value" , "hi" } } ]>

