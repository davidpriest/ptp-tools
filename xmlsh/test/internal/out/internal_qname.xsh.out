spam
prefix:spam
foo:spam
net.sf.saxon.s9api.QName
<bar xmlns="http://foo.bar">hi</bar>
<prefix2:spam xmlns:prefix2="http://foo.bar2" xmlns:prefix1="http://foo.bar1" prefix1:bar="value hi"/>
